package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
This should be "csharp.format.enable".

Reply:
I think it could be phased out now, yes. If I recall correctly, it was added because it wasn't entirely understood how invasive it would be if the extension at one release suddenly started picking up VS Code indentation sizes, after previously ignoring them.